pkgs/tools/text/validator-nu/default.nix

```diff
@@ -10,14 +10,14 @@
 
 let
   pname = "validator-nu";
-  version = "22.9.29";
+  version = "2023-12-18";
```

Comment:
This version string doesn’t follow the [guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming) added in #234201.

Reply:
Thanks, hopefully `23.4.11-unstable-2023-12-18` is OK now. I saw that `23.4.11` was the highest version tagged in the parent commits.